tools/azure-sdk-tools/gh_tools/vnext_issue_creator.py

```diff
@@ -56,6 +60,45 @@ def get_build_link(check_type: CHECK_TYPE) -> str:
         f"https://dev.azure.com/azure-sdk/internal/_build/results?buildId={build_id}&view=logs&j={job_id}&t={next_id}"
     )
 
+def get_build_info(build_link: str, check_type: CHECK_TYPE, service_directory: str, package_name: str) -> str:
+    """Get the build info from the build link."""
+    build_id = os.getenv("BUILD_BUILDID")
+    job_id = os.getenv("SYSTEM_JOBID")
+    timeline_link =f"https://dev.azure.com/azure-sdk/internal/_apis/build/builds/{build_id}/timeline?api-version=6.0"
+
+    if not in_ci():
+        DEVOPS_RESOURCE_UUID = "499b84ac-1321-427f-aa17-267ca6975798"
+        token = DefaultAzureCredential().get_token(f"{DEVOPS_RESOURCE_UUID}/.default").token
+    else:
+        token = os.environ["SYSTEM_ACCESSTOKEN"]
+    AUTH_HEADERS = {"Authorization": f"Bearer {token}"}
+
+    # Make the API request
+    response = requests.get(timeline_link, headers=AUTH_HEADERS)
+    logging.info(f"Response: {response.text}")
+    response_json = json.loads(response.text)
+    # return response_json
+    response_two = []
+    add = False
+    next_id = "b33d1587-3539-5735-af43-e3e62f02ca4b"
+    import gzip
+    import io
+    try:
```

Comment:
let's move this try up a ways... don't want to fail to open an issue if any of the get requests fail